modules/create-object-collections.adoc

```diff
@@ -0,0 +1,80 @@
+====
+
+.Procedure
+To create a deployment collection:
+
+. In the Collections page, click *Create collection*.
+. Enter the name and description for the collection.
+. In the *Collection rules* section, you must perform at least one of the following actions:
+* Define the rules for the collection: See the "Creating collection rules" section for more information.
+*  Attach existing collections to the collection: See the "Adding attached collections" section for more information.
+. The results of your rule configuration or choosing attached collections are available in the *Collection results* live preview panel. Click *Hide results* to remove this panel from display. 
+. Click *Save*. 
+
+[id="creating-collection-rules"]
+== Creating collection rules
+
+When creating collections, you must configure at least one rule or attach another collection to the new one that you are creating. 
+
+[NOTE]
+====
+For {product-title-short} release 3.74, collections are supported only for deployments.
+====
+
+.Procedure
+
+Configure rules to select what resources are included in the collection. Use the preview panel to see the results of the collection rules as you configure them. Rules can be configured in any order.
+
+To configure rules to match deployments:
+
+. In the *Deployments* section, choose one of the following options from the drop-down list.
+
+* *All deployments*: Includes all deployments in the collection. If you choose this option, you must then filter the collection by using namespaces or clusters or by attaching another collection. 
+* *Deployments with names matching* Click this option to select by name and then click one of the following options:
+** Select *An exact value of* and enter the exact name of the deployment.
+** Select *A regex value of* to use regular expression, also called "regex," to search for a deployment. This option is useful if you do not know the exact name of the deployment. A regular expression is a string of letters, numbers, and symbols that defines a pattern. {product-title-short} uses this pattern to match characters or groups of characters and return results. For more information about regular expression, see "Regular-Expressions.info" in the "Additional resources" section.
+* *Deployments with labels matching exactly*: Click this option to select deployments with labels that match the exact text that you enter. The label must be a valid Kubernetes label in the format of `key=value`.
+. Optional: To add more deployments with names or labels that match additional criteria for inclusion, click *OR* and configure another exact or regular expression value.
+
+The following example provides the steps for configuring a collection for a medical application. In this example, you want your collection to include the `reporting` deployment, a database called `patient-db`, and you know that the database is located in a namespace with a label of `medical`. You would perform the following steps:
```

Comment:
Since Kubernetes labels are always of the form `key=value`, I think we should avoid using a language like 'namespaces with a label of medical...' . Instead it would be better to go with 'namespaces with labels matching kubernetes.io/metadata.name=medical' OR 'namespaces with labels where key = kubernetes.io/metadata.name' and value = medical'.